### DISCUSSION TITLE
TypeError: Failed to fetch

### DISCUSSION THREAD

**Discussion by saito-kosuke** [Sanster/IOPaint#196] [General]
I've installed lama-cleaner on my dedicated server.
However, inpainting worked properly on my mac's chrome, but the TypeError: Failed to fetch occurred to me on my windows' chrome as below.
I have no idea why the difference occurred, although the server processing the inpainting is the same.
Could you tell me how to cope with the snag?

<img width="1419" alt="image" src="https://user-images.githubusercontent.com/54809007/214520365-87f69dba-9e00-4a79-b72e-ea3b5660751e.png">

**Comment by saito-kosuke** [comment]
Another error(Not allowed to request resource) has occurred to me on my mac's safari.
Is there a browser/os limitation where we can use your service?
![image](https://user-images.githubusercontent.com/54809007/214522113-e1fc7d20-606a-4701-b400-cb63920f53d7.png)

**Comment by Sanster** [comment]
Please check the server-side error log

**Reply by saito-kosuke** [reply]
i found a mistake regarding REACT_APP_INPAINTING_URL
thank you so much
you can close the discussion
